src/Laravel/Eloquent/Metadata/ModelMetadata.php

```diff
@@ -131,13 +131,14 @@ public function getVirtualAttributes(Model $model, array $columns): Collection
                 'name' => $name,
                 'type' => null,
                 'increments' => false,
-                'nullable' => null,
+                'nullable' => true,
```

Comment:
The `nullable` field [must be a boolean](https://github.com/symfony/symfony/blob/8debb252592cf7d6943ac34968413d1a9a7b501d/src/Symfony/Component/PropertyInfo/Type.php#L77), as it is given to the `new Type()` here: https://github.com/api-platform/core/blob/80fc2d2303f35bccdc6789dc72a205781c3dc3d0/src/Laravel/Eloquent/Metadata/Factory/Property/EloquentPropertyMetadataFactory.php#L73-L81